Makefile

```diff
@@ -7,7 +7,13 @@ XYZ = node_modules/.bin/xyz --repo git@github.com:sanctuary-js/sanctuary.git --s
 
 
 .PHONY: all
-all: README.md
+all: LICENSE README.md
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Sanctuary/Copyright (c) $(shell git log --date=format:%Y --pretty=format:%ad | sort -r | head -n 1) Sanctuary/' '$@.orig' >'$@'
```

Comment:
I did consider `-1`. It's hypothetically possible, though, that the HEAD commit was authored a year ago but only recently pushed to `master`. This is an unlikely scenario, but since I'm about to open similar pull requests in several repositories I thought it worth writing a pipeline which handles this edge case.

`git log` provides an `--author-date-order` flag which would obviate the need for `sort`, but I find `sort` clearer.